test/functionalTest/cases/1316_create_subscription/post_subs_errors.test

```diff
@@ -247,41 +247,41 @@ echo
 01. POST /v2/subscriptions (without id/idpattern)
 =================================================
 HTTP/1.1 400 Bad Request
-Content-Length: 94
+Content-Length: 96
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "subject entities element does not have id nor idPattern",
+    "description": "/entities/ vector item does not have /id/ nor /idPattern/",
```

Comment:
These error messages were changed as the *parse function* was made generic. Before it was just to parse subscriptions, not it parses registrations as well.

Reply:
It wouldn't be the first time we adjust description messages in error responses. This is not considered backward compability problem, as description (different from error field or HTTP response code) is aimed for humans, not machines.

Reply:
ok, great, no probs thus

Reply:
NTC